src/psyclone/psyad/lfric/Makefile

```diff
@@ -0,0 +1,54 @@
+TL_DIR = tangent_linear
+TL_TWEAKED_DIR = tangent_linear_tweaked
+ADJ_PARTIAL_DIR = adjoint_partial
+
+all: ${ADJ_PARTIAL_DIR}/adj_dg_inc_matrix_vector_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_moist_dyn_mass_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_project_eos_pressure_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_hydrostatic_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_transpose_matrix_vector_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_kinetic_energy_gradient_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_vorticity_advection_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_moist_dyn_gas_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_pressure_gradient_bd_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_matrix_vector_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_rhs_project_eos_kernel_mod.F90 ${ADJ_PARTIAL_DIR}/adj_rhs_sample_eos_kernel_mod.F90
```

Comment:
These files don't exist in the repo, they are generated so no I don't think so. The only way would be to take the files from the TL_DIR and TL_TWEAKED_DIR, rename them and then change their directories. Is this possible in a Makefile?

Reply:
Yes, since we've already gone down the road of using Gnu make you can use `subst` for this I think, e.g. `ADJ_KERNEL_F90_FILE = $(subst tl_,adj_,${TL_KERNEL_FILE})`

Reply:
They are lovely now :-)